dasharo-security/sort-pci-semiauto.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
+Library             Collections
+Library             DateTime
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Variables ***
+${LSPCI_CMD}=       lspci -vv
+${IP_LINK_CMD}=     ip link
```

Comment:
It doesn't make sense to put those commands into variables. It only obfuscates what is really done by the test. It would be justified if it was some really long command.

Reply:
updated test https://github.com/Dasharo/open-source-firmware-validation/pull/515/commits/b8b54f5ba57824d5099809577c355ff6c8daf6cb